pkg/reconciler/messaging/channel/channel_test.go

```diff
@@ -361,42 +361,26 @@ func TestAllCases(t *testing.T) {
 				Eventf(corev1.EventTypeWarning, "SubscriberNotOwned", "Subscriber %q is not owned by this channel", "cre-sub-testsubscription-abc-123"),
 				Eventf(corev1.EventTypeWarning, reconciledSubscribersFailedReason, "Reconcile Subscribers failed with: channel %q does not own subscriber %q", channelName, "cre-sub-testsubscription-abc-123"),
 			},
-			WantStatusUpdates: []clientgotesting.UpdateActionImpl{{
```

Comment:
@nachocano 
I figured out the issue
We never update status in this test case since we use `equality.Semantic.DeepEqual` to compare the status
https://github.com/google/knative-gcp/blob/94e9331b180d315d058ea867b8e368096f807e80/pkg/client/injection/reconciler/messaging/v1beta1/channel/reconciler.go#L184
I ran some example,
for a, b and c, all of them are equality.Semantic.DeepEqual
`func TestEqual(t *testing.T) {
	a:= eventingduckv1beta1.SubscribableStatus{}
	b:= eventingduckv1beta1.SubscribableStatus{Subscribers:[]eventingduckv1beta1.SubscriberStatus{}}
	c:= eventingduckv1beta1.SubscribableStatus{Subscribers:[] eventingduckv1beta1.SubscriberStatus(nil)}
	fmt.Println(equality.Semantic.DeepEqual(a, b))
	fmt.Println(equality.Semantic.DeepEqual(a, c))
	fmt.Println(equality.Semantic.DeepEqual(b, c))
}`